openshift/compositions/mailhog-deploymentconfig.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: DeploymentConfig
+metadata:
+  annotations:
+    kompose.cmd: kompose convert --provider=openshift --build=deploy-config --build-repo=https://github.com/bcgov/moh-prime.git
+    kompose.version: 1.18.0 (06a2e56)
+  creationTimestamp: null
+  labels:
+    io.kompose.service: mailhog-server
+  name: mailhog-server
+spec:
+  replicas: 1
+  selector:
+    io.kompose.service: mailhog-server
+  strategy:
+    resources: {}
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        io.kompose.service: mailhog-server
+    spec:
+      containers:
+      - image: ' '
+        name: mailhog-server
+        ports:
+        - containerPort: 1025
+        - containerPort: 1025
```

Comment:
Is this duplication intentional?

Reply:
Pretty sure it is - Kompose/Kubernetes built this source template.  I think it needs it for proper mapping (I'm opening 25 and 1025 to the same port on the container)

Reply:
Ok, that makes sense. It is in the DC file as well, can I assume its the same there?